newIDE/app/src/AnnouncementsFeed/AnnouncementsFeed.js

```diff
@@ -0,0 +1,100 @@
+// @flow
```

Comment:
should the file be renamed index since it has the same name as the folder?

Reply:
Yes!

Reply:
Yes, done!